misc-tools/dj_make_chroot.in

```diff
@@ -147,6 +129,29 @@ if [ "$(id -u)" != 0 ]; then
 fi
 
 [ -z "$CHROOTDIR" ] && error "No chroot directory given or default known."
+
+# If host system is Debian or Ubuntu, use its release and architecture by default
```

Comment:
I think it makes more sense to put this block a couple of lines up, after the DISTRO/RELEASE checks, before the root check.